services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -488,15 +488,46 @@ async def try_update_database_from_storage(
     )
     async def auto_update_database_from_storage_task(
         self, file_uuid: str, bucket_name: str, object_name: str
-    ):
+    ) -> Optional[FileMetaDataEx]:
         return await self.try_update_database_from_storage(
             file_uuid, bucket_name, object_name, silence_exception=True
         )
 
-    async def upload_link(self, user_id: str, file_uuid: str):
+    async def update_metadata(self, file_uuid: str) -> Optional[FileMetaDataEx]:
+        bucket_name = self.simcore_bucket_name
+        object_name = file_uuid
+        return await self.auto_update_database_from_storage_task(
+            file_uuid=file_uuid,
+            bucket_name=bucket_name,
+            object_name=object_name,
+        )
+
+    async def delete_metadata(self, user_id: int, file_uuid: str) -> None:
+        async with self.engine.acquire() as conn:
+            can: Optional[AccessRights] = await get_file_access_rights(
+                conn, int(user_id), file_uuid
+            )
+            if not can.write:
+                message = f"User {user_id} was not allowed to upload file {file_uuid}"
+                logger.debug(message)
+                raise web.HTTPForbidden(reason=message)
+
+            try:
+                await conn.execute(
+                    file_meta_data.delete().where(
+                        file_meta_data.c.file_uuid == file_uuid
+                    )
+                )
+            except Exception:
+                message = f"Could not delete metada entry for file {file_uuid}"
```

Comment:
spelling:
metadata.

Also, from a devops point of view: This message should probably trigger some sort of alert, because it spoils the state of our storage? I mean if this happens the state of the assoc. project might be fishy

Feel free to highlight also which characteristic messages should alert ops people in your PR, we'll take care of it during the devops changes